### PR TITLE
tests: do not ./ceph* to obey PATH

### PR DESCRIPTION
**Pull request by dachary**
Signed-off-by: Loic Dachary <ldachary@redhat.com>

### CODE REVIEW

**Comment by dachary** [comment]
@alimaredia let see if it pleases the make check bot :-)

File: src/unittest_bufferlist.sh [hunk 1]
```diff
@@ -16,4 +16,4 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU Library Public License for more details.
 #
-CEPH_BUFFER_TRACK=true ./unittest_bufferlist
+CEPH_BUFFER_TRACK=true unittest_bufferlist
```
Comment (tchaikov) [inline src/unittest_bufferlist.sh]: @dachary did you changed the `$PATH` for `make check`? one of failures is
```
./unittest_bufferlist.sh: line 19: unittest_bufferlist: command not found
```

**Comment by dachary** [comment]
@tchaikov updated the PATH for both unittest_bufferlist and ceph_objectstore_tool tests and repushed, thanks for your review :-)

**Comment by dachary** [comment]
@alimaredia are these changes helpful to you ?

**Pull request closed without merging**